hoard/sources/whoas.py

```diff
@@ -135,7 +135,14 @@ def create_from_whoas_dim_xml(data: str) -> Dataset:
             and "qualifier" in field.attrib
             and field.attrib["qualifier"] == "ispartof"
         ):
-            kwargs["series"] = Series(seriesName=field.text)
+            if field.text is not None and field.text.startswith(
+                "https://hdl.handle.net/"
+            ):
+                series_args = {"seriesInformation": field.text}
+                id = f"oai:darchive.mblwhoilibrary.org:{field.text[23:]}"
```

Comment:
Using a numeric index here makes me nervous. If the URL gets passed as http instead of https then this breaks. Since it's a URL, the safer way to get what you want will be to use  `urlparse()` from `urllib.parse`.